src/main/kotlin/no/nav/syfo/varsel/naermesteleder/NaermesteLederVarselType.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.syfo.varsel.naermesteleder
+
+enum class NaermesteLederVarselType(val id: String) {
+    NAERMESTE_LEDER_MOTE_PLANLAGT("syfoDialogmotePlanlagt"),
+    NAERMESTE_LEDER_MOTE_AVLYST("syfoDialogmoteAvlyst"),
+    NAERMESTE_LEDER_MOTE_NYTID("syfoDialogmoteNytid"),
+}
```

Comment:
Pirk: Generelt rundt bruk av æ,ø,å, så har det tidligere blitt snakket om at vi ikke vil skrive ae, oe, aa for disse, men heller bare bruke en bokstav som erstatter de.